inference-engine/tests/functional/inference_engine/transformations/mul_fake_quantize_fusion.cpp

```diff
@@ -218,7 +251,7 @@ TEST(TransformationTests, MulFakeQuantizeFusionConstantSomeNegative) {
     ASSERT_TRUE(res.first) << res.second;
 }
 
-TEST(TransformationTests, MulFakeQuantizeFusionConstantSomeNegativeF16) {
+TEST(TransformationTests, DISABLED_MulFakeQuantizeFusionConstantSomeNegativeF16) {
```

Comment:
I think we need either remove fp16 tests or updated them to use fp32. Because this test will never be fixed.